vins_estimator/src/estimator_node.cpp

```diff
@@ -11,38 +11,8 @@
 #include "estimator.h"
 #include "parameters_ros.h"
 #include "utility/visualization.h"
-
+#include <memory>
 Estimator estimator;
```

Comment:
Is there any reason why estimator must be a global here? I see all the calls are within the EstimatorNode class, maybe it could be a private member

Reply:
Nope. Will update :D

Reply:
Magically, this might causes a consistent segfault upon starting the node... Still investigating...but wow